tests/manifests/open_api/test_openapi.py

```diff
@@ -0,0 +1,31 @@
+import json
+from pathlib import Path
+
+from spinta.core.config import RawConfig
+from spinta.testing.manifest import load_manifest
+
+
+def test_open_api_manifest(rc: RawConfig, tmp_path: Path):
+    # TODO: Temporary test, will be adjusted once actual logic is implemented.
+    data = json.dumps({
+            "openapi": "3.0.0",
+            "info": {
+            "title": "openapi",
+            "version": "1.0.0"
+        },
+        "paths": {}
+    })
+
+    table = '''
+        id | d | r | b | m | property | type | ref | source | prepare | level | access | uri | title | description
+           | openapi                  |      |     |        |         |       |        |     |       |
```

Comment:
```suggestion
           | countries_api                  |      |     |        |         |       |        |     |       |
```

+ title, description

Reply:
[4dfbd9d](https://github.com/atviriduomenys/spinta/pull/1212/commits/4dfbd9d24a08d590219472784fd669633cf23e7a)